src/commands/api-mesh/create.js

```diff
@@ -81,7 +81,7 @@ class CreateCommand extends Command {
 				let sdkList = [];
 
 				if (mesh) {
-					this.log('Successfully created mesh %s', mesh.meshId);
+					this.log('Your mesh is being processed. Please wait 5 minutes before checking the status of your mesh %s', mesh.meshId);
```

Comment:
Wording needs a review from Jared/Cris

Reply:
I think 5 minutes is too much time to wait

Reply:
Nishant said 5 minutes was fine to let customers know. Also having meshes close to instantly available shouldn't be expectations that we set for customers.

Reply:
Updated wording @prabhuram93